Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -45,21 +49,32 @@ final class SettingsViewModel: ObservableObject {
         }
     }
     
-    func addVehicle(_ vehicle: Vehicle) async {
-        
+    /// Adds a new vehicle to the Firestore database and the local 'vehicles' array.
```

Comment:
```suggestion
    /// Adds a new vehicle to the Firestore database and the local `vehicles` array.
```

Check all your documentation comments, anything relating to code objects should use single or double backticks where appropriate, it looks like you used apostrophes